subprojects/test-kit/src/integTest/groovy/org/gradle/testkit/runner/GradleRunnerDeprecatedBuildJvmIntegrationTest.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testkit.runner
+
+import org.gradle.integtests.fixtures.AvailableJavaHomes
+import org.gradle.testkit.runner.fixtures.NoDebug
+import org.gradle.testkit.runner.fixtures.NonCrossVersion
+import spock.lang.IgnoreIf
+
+@NonCrossVersion
+class GradleRunnerDeprecatedBuildJvmIntegrationTest extends BaseGradleRunnerIntegrationTest {
+
+    @IgnoreIf({ AvailableJavaHomes.jdk7 == null })
+    @NoDebug
```

Comment:
:x: It would make sense to also test this with TestKit's debug mode.

Reply:
Sure.

Reply:
Done.